src/dune_rules/watermarks.ml

```diff
@@ -146,7 +146,12 @@ module Dune_project = struct
   let filename = Path.in_source Dune_project.filename
 
   let load ~dir ~files ~infer_from_opam_files =
-    let project = Dune_project.load ~dir ~files ~infer_from_opam_files in
+    let project =
+      (* XXX dir_status only affects warning status, but it will not matter
```

Comment:
```suggestion
      (* dir_status only affects warning status, but it will not matter
```

Reply:
To be fair, in this case the code is clear enough with a comment; `Normal` is the value I would expect in `dune subst`. It would be weird for `dune subst` to pass any other value here